src/notation/view/notationcontextmenumodel.cpp

```diff
@@ -200,6 +202,40 @@ MenuItemList NotationContextMenuModel::makeElementItems()
         items << makeMenuItem("edit-element");
     }
 
+    items << makeSeparator();
+
+    MenuItem* item = new MenuItem(this);
+
+    mu::ui::UiAction action;
+    action.code = "edit-style";
+    action.title = TranslatableString("notation", "Style...");
+    item->setAction(action);
+
+    QString id = QString::fromStdString(item->action().code) + QString::number(0);
```

Comment:
You won't need this code after resolving this comment (see this constructor MenuItem::MenuItem(const UiAction& action, QObject* parent))
https://github.com/mathesoncalum/MuseScore/pull/6/files#r1312833309